ballerina/LICENSE

```diff
@@ -12,175 +12,175 @@ At the bottom of this file is a table that shows what each license indicated
 below is and where the actual text of the license can be found.
 
 
-Name                                                                                                Type        License             
+Name                                                                                                Type           License             
 _________________________________________________________________________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2             
 commons-text-1.10.0.jar                                                                             bundle         apache2             
 compiler-0.8.9.jar                                                                                  jar            apache2             
-stream-2201.6.1.jar                                                                                 jar            apache2             
-jballerina.java-2201.6.1.jar                                                                        jar            apache2             
-gson-2.8.9.jar                                                                                      jar            apache2             
+future-2201.6.2.jar                                                        jar            apache2             
```

Comment:
shall we align the `jar` and `apache2` parts with other lines?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/4979/commits/e7af3b1cb1930e4b240bd1d988e3eebddb7d5aa8